sklearn/model_selection/_validation.py

```diff
@@ -343,6 +346,32 @@ def _normalize_score_results(scores, scaler_score_key="score"):
     return {scaler_score_key: scores}
 
 
+def _warn_about_fit_failures(results, error_score):
+    fit_errors = [
+        result["fit_error"] for result in results if result["fit_error"] is not None
+    ]
+    num_fits = len(results)
+
+    if fit_errors:
+        num_failed_fits = len(fit_errors)
+        fit_errors_counter = Counter(fit_errors)
+        delimiter = "-" * 80 + "\n"
+        fit_errors_summary = "\n".join(
+            f"{delimiter}{n} fits failed with the following error:\n{error}"
+            for error, n in fit_errors_counter.items()
+        )
+
+        some_fits_failed_message = (
+            f"\n{num_failed_fits} fits failed on the training sets over a total of"
```

Comment:
I find this language a bit strange... Should "over" be "out of"? Is "on the training sets" helpful?

Reply:
Thanks for the comment, I updated the wording:
- now: "5 fits failed out of a total of 15"
- before it was: "5 fits failed on the training sets over a total of 15 fits"

Let me know if the language is still a bit strange!